src/erlcloud_s3.erl

```diff
@@ -467,9 +468,17 @@ extract_acl(ACL) ->
     [extract_grant(Item) || Item <- ACL].
 
 extract_grant(Node) ->
-    [{grantee, extract_user(xmerl_xpath:string("Grantee", Node))},
+    [{grantee, extract_grantee(hd(xmerl_xpath:string("Grantee", Node)))},
      {permission, decode_permission(erlcloud_xml:get_text("Permission", Node))}].
 
+extract_grantee(Node) ->
```

Comment:
It doesn't answer my question: what is it for and what problem are we trying to solve? aws responses validation or anything else?